ib-cli/README.md

```diff
@@ -82,9 +82,14 @@ export AUTH_FILE=/path/to/seed-image-pull-secret.json
 export PS_FILE=/path/to/release-pull-secret.json
 export SSH_PUBLIC_KEY=~/.ssh/id_rsa.pub
 export IBI_INSTALLATION_DISK=/dev/sda
+# Start point of the /var/lib/containers partition. Free space before it will be allocated to system partition
+# It can be specified in a positive manner: 120G , meaning that new partition will start at 120Gb of disk, and extend to the end of the disk
```

Comment:
While it's nice do users care whether the extra partition is at the end or at the start of the disk?
If not perhaps asking for the partition size is more simple

Reply:
I like the "give me the size" approach, but it differs from the way you define it for the seed

While creating the seed cluster, you specify the start of the partition. and AFAIK you cannot say "I want a partition of 5Gb at the end of the disk"

In IBI case, since we use sgdisk, we can just provide a negative number as the start of the partition, and it counts backwards since the end of the disk, so effectively is the size of the partition

My prefered approach is to just specify the desired size of the partition, and prepend it by - , but I thought better to give context to the user, since partitioning in the end "is not part" of LCA and lands more on user side ...